pkg/kubelet/dockertools/manager.go

```diff
@@ -949,18 +946,33 @@ func (dm *DockerManager) checkVersionCompatibility() error {
 	return nil
 }
 
-func (dm *DockerManager) getDefaultSecurityOpt() ([]string, error) {
+func (dm *DockerManager) getSecurityOpt(container *api.Container) ([]string, error) {
```

Comment:
Look for the annotation on the pod for this container (needs to be changed to accept the pod)

The precedent should be:

1.  container-specific annotation, if any
2.  pod-level annotation, if any
3.  default (unconfined?)